Assets/Scripts/UI/Canvas/PhotoTaking/CameraPanel.cs

```diff
@@ -0,0 +1,19 @@
+using UnityEngine;
+using TMPro;
+
+using Framework.ScriptableObjects;
+
+namespace UI.Canvas.PhotoTaking
+{
+    public class CameraPanel : MonoBehaviour
+    {
+        [SerializeField] private TextMeshProUGUI buildingName;
```

Comment:
```suggestion
        [SerializeField] private TMP_Text buildingName;
```
Deze varible is iets duidelijker wat het is. Weet niet eens waar `UGUI` voor staat.